airflow/www/extensions/init_views.py

```diff
@@ -220,7 +224,7 @@ def resolve(self, operation):
         return _LazyResolution(self.resolve_function_from_operation_id, operation_id)
 
 
-class _CustomErrorRequestBodyValidator(RequestBodyValidator):
+class _CustomErrorRequestBodyValidator(AbstractRequestBodyValidator):
```

Comment:
This behavior was contributed to Connexion upstream and is included in Connexion 3. A custom validator is no longer needed.